### ISSUE TITLE
Date Roller Prefetch issues

### ISSUE THREAD

**Issue by c-schuler** [cqframework/cqf-tooling#385]
The date roller will set the prefetch to null if it doesn't exist, but then throws an error if the prefetch is null. So it is returning data in a format that is throwing an error when re-running the operation.

It also doesn't seem to like Bundles in the prefetch. Getting:
```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "com.google.gson.JsonObject.toString()" because the return value of "com.google.gson.JsonObject.getAsJsonObject(String)" is null
	at org.opencds.cqf.tooling.dateroller.HookDataDateRoller.rollPrefetchItemsDates(HookDataDateRoller.java:74)
	at org.opencds.cqf.tooling.dateroller.HookDataDateRoller.rollJSONHookDates(HookDataDateRoller.java:36)
	at org.opencds.cqf.tooling.dateroller.DataDateRollerOperation.rollDatesInFile(DataDateRollerOperation.java:101)
	at org.opencds.cqf.tooling.dateroller.DataDateRollerOperation.processFiles(DataDateRollerOperation.java:78)
	at org.opencds.cqf.tooling.dateroller.DataDateRollerOperation.processFiles(DataDateRollerOperation.java:72)
	at org.opencds.cqf.tooling.dateroller.DataDateRollerOperation.rollAllDates(DataDateRollerOperation.java:65)
	at org.opencds.cqf.tooling.dateroller.DataDateRollerOperation.execute(DataDateRollerOperation.java:57)
```
Runs fine when Bundle is removed.

**Comment by c-schuler** [comment]
Also looks like it is removing doseQuantity from MedicationRequests?

**Comment by c-schuler** [comment]
Resolved in #386
